tests/python/unittest/test_loss.py

```diff
@@ -342,12 +343,45 @@ def test_triplet_loss():
     Loss = gluon.loss.TripletLoss()
     loss = Loss(output, pos, neg)
     loss = mx.sym.make_loss(loss)
-    mod = mx.mod.Module(loss, data_names=('data',), label_names=('pos','neg'))
+    mod = mx.mod.Module(loss, data_names=('data',), label_names=('pos', 'neg'))
     mod.fit(data_iter, num_epoch=200, optimizer_params={'learning_rate': 0.01},
             initializer=mx.init.Xavier(magnitude=2), eval_metric=mx.metric.Loss(),
             optimizer='adam')
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.05
 
+@with_seed()
+def test_poisson_nllloss():
```

Comment:
Please separate into 3 unit tests clearly stating what condition are you testing.

Reply:
@Lai Yes, I have separated the use cases commenting on what exactly is being tested. I have not split into different functions so as to keep consistency with other loss functions tests.
Thank you for your comment.